main.go

```diff
@@ -43,88 +49,106 @@ func initClientConnection() bpb.BirthdayFunctionsClient {
 		log.Fatalln("connection error: ", err)
 		os.Exit(4)
 	}
-	// defer conn.Close()
 
 	client := bpb.NewBirthdayFunctionsClient(conn)
 
 	return client
 }
 
-//CreateBirthday inserts a birthday object
-func (r *Router) CreateBirthday(c *gin.Context) {
+func corsRouterConfig() cors.Config {
+	corsConfig := cors.DefaultConfig()
+	corsConfig.AddExposeHeaders("x-uploadid")
+	corsConfig.AllowAllOrigins = false
+	corsConfig.AllowWildcard = true
+	corsConfig.AllowOrigins = strings.Split("http://localhost*", ",")
+	corsConfig.AllowCredentials = true
+	corsConfig.AddAllowHeaders(
+		"x-content-length",
+		"authorization",
+		"cache-control",
+		"x-requested-with",
+		"content-disposition",
+		"content-range",
+		"destination",
+		"fileID",
+	)
+
+	return corsConfig
+}
 
+// CreateBirthday inserts a birthday object if it
+// doesn't exist. if it does, its being overritten
+func (r *Router) CreateBirthday(c *gin.Context) {
 	request := &bpb.CreateBirthdayRequest{
-		PersonalNumber: c.Request.FormValue(personalNumber),
-		Name:           c.Request.FormValue(name),
-		Date:           c.Request.FormValue(date),
+		PersonalNumber: c.Request.FormValue(PersonalNumber),
+		Name:           c.Request.FormValue(Name),
+		Date:           c.Request.FormValue(Date),
 	}
-
 	res, err := r.client.CreateBirthday(c, request)
 	if err != nil {
-		fmt.Println("create birthday error: ", err)
+		log.Fatal("create birthday error: ", err)
```

Comment:
it's not fatal

Reply:
u must understand that once you call 'fatal', os.exit  is called

Reply:
done